src/components/Form/Readme.mdx

```diff
@@ -97,27 +160,74 @@ Checkbox uses [rc-checkbox](https://github.com/react-component/checkbox) under t
 
 <Props of={Checkbox} />
 
+Checkbox will expose an `event.target` with
+* `name`
+* `type` as `checkbox`
+* `value` as a boolean indicating the `checked` value
+* `checked` as a boolean indicating the `checked` value
+
 <Playground>
   {() => {
-    const [isVegan, setIsVegan] = useState(false);
-    const [isDogPerson, setIsDogPerson] = useState(false);
+    const [formData, setFormData] = useState({});
+    const [lastEvent, setLastEvent] = useState({});
+    const onChange = (e) => {
+      const { name, value } = e.target;
+      setFormData({ ...formData, [name]: value })
```

Comment:
you're not setting `lastEvent` and the event demo box stays empty....

Reply:
Oops, will fix that!